src/Apps/BugBashPro/scripts/Hubs/BugBashDirectory/Components/BugBashDirectoryTable.tsx

```diff
@@ -88,10 +88,10 @@ function getColumns(
             isSortedDescending: isSortedDescending,
             renderCell: (_rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IBugBash>, bugBash: IBugBash) => {
                 return (
-                    <SimpleTableCell columnIndex={columnIndex} tableColumn={tableColumn} key={`col-${columnIndex}`}>
+                    <SimpleTableCell className="bolt-table-cell-content-with-link" columnIndex={columnIndex} tableColumn={tableColumn} key={`col-${columnIndex}`}>
                         <AsyncLinkComponent
                             key={bugBash.id}
-                            className="text-ellipsis bolt-table-link bolt-table-inline-link flex-grow"
+                            className="text-ellipsis bolt-table-link"

```

Comment:
I would recommend taking off the flex-grow, but having the onActivate of the <Table /> call the the same onClick. That way, clicking anywhere in the table will also navigate you.

Reply:
And also using singleClickActivation maybe?

Reply:
I'll try that. The reason i was using flex-grow was because the Title column is too wide and the link's hit-target is too small compared to the cell width.